Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -52,5 +54,25 @@ public async Task<IEnumerable<WordOfTheDayStatsDto>> GetWordOfTheDayStats(DateTi
         {
             return (await _wordService.GetWordOfTheDayStatsAsync(date, days, playerId ));
         }
+
+        [HttpGet("WordList")]
+        public async Task<IEnumerable<Word>> GetWordList(int pageNumber = 1, int pageLength = 20, string? searchString = null)
+        {
+            return await _wordService.GetWordList(pageNumber, pageLength, searchString);
+        }
+
+        [HttpGet("DeleteWord")]
+        [Authorize(Policy = Policies.CanCreateDeleteWords)]
+        public async Task<bool> DeleteWord(int wordId)
+        {
+            return await _wordService.DeleteWord(wordId);
+        }
+
+        [HttpGet("ChangeCommonFlag")]
+        [Authorize(Policy = Policies.CanCreateDeleteWords)]
```

Comment:
Anyone logged in should be able to update the common word flag, so it should be a different policy